### PR TITLE
AP-5426 Create capital disregards DB table and add namespace

### PR DESCRIPTION
**Pull request by RoseSAK**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-5426)

Add a new table `CapitalDisregards` to record both mandatory and discretionary disregards.
Add the `capital_disregards` namespace 

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- The standards in the [Git Workflow document on Confluence](https://dsdmoj.atlassian.net/wiki/spaces/ATPPB/pages/4602855954/Git+Workflow) should be followed
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -427,6 +427,19 @@
     t.index ["legal_aid_application_id"], name: "index_dependants_on_legal_aid_application_id"
   end
 
+  create_table "disregards", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```
Comment (jsugarman) [inline db/schema.rb:430]: should we not be calling this policy_disregards to avoid confusion with benefit disregards?

Comment (jsugarman) [reply db/schema.rb:430]: or perhaps `capital_disregards` (if it makes business sense) if we are avoiding the `policy_regards` name due to the other table

Comment (RoseSAK) [reply db/schema.rb:430]: Hmm, yeah probably, we already have an existing PolicyDisregards table though which we were going to leave for a while until we migrate the data. Perhaps we'd need to rename this to `LegacyPolicyDisregards`? Or alternatively migrate all the old policy disregards data in this PR. 

@kmahern and @skdejong what do you guys think?

Comment (skdejong) [reply db/schema.rb:430]: I think `capital_disregards` would make the most sense out of all those options.

Comment (kmahern) [reply db/schema.rb:430]: I think I agree with @skdejong that capital_disregards might be the best option. For consistency should we also make the namespace capital_disregards?

Comment (skdejong) [reply db/schema.rb:430]: Yes, I think we should keep things consistent.

**Review by colinbruce — APPROVED** [review]

**Pull request merged**